template_lib.py

```diff
@@ -88,6 +89,19 @@ def _render_cmd(code: str) -> str:
 
 
 class Renderer(markdown_code_blocks.CodeRenderer):
+    def link(
+        self, link: str, text: Optional[str], title: Optional[str],
+    ) -> str:
+        match = NAMED_ANCHOR_RE.search(link)
+        if match:
+            a_id = match.group(1)
+            return (
+                f'<a name="{a_id}"></a>'
+                f'<a href="#{a_id}">{title}</a>'
```

Comment:
does `<a name="{...} href="#{...}">...</a>` work?

Reply:
Turns out it does! I was copying the approach from Terraform documentation ([example](https://www.terraform.io/docs/providers/aws/r/instance.html#cpu_core_count)) where they have them separated, but yup, it works combined - have changed it